### ISSUE TITLE
Runtime exception when requiring flaschenpost

### ISSUE THREAD

**Issue by colorizedmind** [thenativeweb/flaschenpost#18]
After requiring flaschenpost the following runtime exception occurs:

```
module.js:557
    throw err;
    ^

Error: Cannot find module 'babel-runtime/helpers/classCallCheck'
    at Function.Module._resolveFilename (module.js:555:15)
    at Function.Module._load (module.js:482:25)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Projects\pb\node_modules\flaschenpost\dist\Configuration\index.js:3:24)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
```

With flaschenpost 1.0.0 everything works as expected. The exception occurs only with the new 1.1.0 version. I'm using Node.js 9.5.0 on Windows 10.

### Steps to reproduce

* Require flaschenpost in one of your source files.

### Expected result

* The flaschenpost dependency is loaded without any error.

### Actual result

* The exception pasted above occurs.

**Comment by goloroden** [comment]
Hey @colorizedmind 😊 

Thanks for the feedback, and for bringing up this issue!

Could you please try to directly install flaschenpost from GitHub, from the [updated-package-lock branch](https://github.com/thenativeweb/flaschenpost/tree/updated-package-lock)?

The entry in your `package.json` should look like this:

```
"flaschenpost": "thenativeweb/flaschenpost#updated-package-lock"
```

Does the problem then persist?

**Comment by colorizedmind** [comment]
After changing the entry in the `package.json` and running `npm install` again I've got still the same problem than before. The `package-lock.json` lists the dependency as follows:

```json
"flaschenpost": {
  "version": "github:thenativeweb/flaschenpost#170f46146a4a3ee513764d033b2aea20ca1e073c",
  "requires": {
    "app-root-path": "2.0.1",
    "chalk": "2.3.1",
    "find-root": "1.1.0",
    "lodash": "4.17.5",
    "moment": "2.20.1",
    "processenv": "1.0.0",
    "split2": "2.2.0",
    "stack-trace": "0.0.10",
    "stringify-object": "3.2.2",
    "untildify": "3.0.2",
    "varname": "2.0.3"
  }
}
```

**Comment by goloroden** [comment]
@grundhoeferj Can you please have a look at this, since you can do this on Windows?

**Comment by colorizedmind** [comment]
Nevermind, I just mentioned that I don't have access to the branch you linked before. Maybe it's set to private?

**Comment by goloroden** [comment]
No, I had deleted it, since it didn't solve the problem.

**Comment by grundhoeferj** [comment]
Hey @colorizedmind 😊 
i could reproduce the error. There is now a new version available: `flaschenpost@1.1.1`

Can you give me a short feedback please, if it works on your machine? After that i will close this issue.

**Comment by colorizedmind** [comment]
With the added babel-runtime dependency it's working now on my machine. Thank you.
